assets/js/stripe-admin.js

```diff
@@ -91,10 +91,10 @@ jQuery( function( $ ) {
 				e.preventDefault();
 
 				// Remove any previous messages.
-				$( '.wc-stripe-apple-pay-domain-message' ).remove();
+				$( '.wc-stripe-apple-pay-domain-message' ).text( '' );
```

Comment:
It is not good practice to blank out a DOM element leaving it sit there hidden with no purpose. It is better to remove and add it when need it.